src/vs/workbench/parts/extensions/electron-browser/extensionsActions.ts

```diff
@@ -1156,18 +1157,60 @@ export class DisableAction extends Action {
 export class CheckForUpdatesAction extends Action {
 
 	static readonly ID = 'workbench.extensions.action.checkForUpdates';
-	static LABEL = localize('checkForUpdates', "Check for Updates");
+	static LABEL = localize('checkForUpdates', "Check for Extension Updates");
 
 	constructor(
 		id = UpdateAllAction.ID,
 		label = UpdateAllAction.LABEL,
-		@IExtensionsWorkbenchService private extensionsWorkbenchService: IExtensionsWorkbenchService
+		@IExtensionsWorkbenchService private extensionsWorkbenchService: IExtensionsWorkbenchService,
+		@IViewletService private viewletService: IViewletService,
+		@INotificationService private notificationService: INotificationService
 	) {
 		super(id, label, '', true);
 	}
 
+	private onUpdateNotAvailable(): void {
+		this.notificationService.notify(
+			{
+				severity: severity.Info,
+				message: localize('noUpdatesAvailable', "All Extensions are up to date.")
+			});
+	}
+
+	private onUpdateAvailable(outdatedCount: number): void {
+		this.notificationService.notify(
+			{
+				severity: severity.Info,
+				message: localize('updatesAvailable', "{0} extensions are outdated.", outdatedCount)
```

Comment:
`extensions updates found` sounds more positive than `extensions are outdated`  :)
Also account for the case where only 1 extension is outdated. You will have to use 2 separate msgs.